website/static/css/custom.css

```diff
@@ -151,4 +151,25 @@ pre code.hljs.css {
 .docMainWrapper .mainContainer .wrapper .post h5,
 .docMainWrapper .mainContainer .wrapper .post h6 {
   font-weight: bold;
+}
+
+@media screen and (max-width: 767px) {
+  .hide-in-mobile {
+    display: none;
+  }
+  .home-getting-started {
+    width: 100%;
+  }
+
+  .buttons-unit {
+    margin: 30px 0 60px;
+  }
+
+  .hero .text,
+  .hero  .minitext,
```

Comment:
Nit - Double space between selectors.

Reply:
Turned off auto-format and forgot to turn it on 🤦🏻‍♂️ Thanks for noticing!